e2e/tests/api_tests/lrspec/api_2v1_spec_test.js

```diff
@@ -70,6 +70,12 @@ Scenario('Settle claim 2v1 scenario', async ({I, api_spec}) => {
   await api_spec.settleClaimSelectClaimant(config.applicantSolicitorUser, 'YES');
 });
 
+Scenario('Discontinue claim 2v1 scenario', async ({I, api_spec}) => {
+  await api_spec.createClaimWithRepresentedRespondent(config.applicantSolicitorUser, 'TWO_V_ONE');
+  await api_spec.discontinueClaim(config.applicantSolicitorUser);
+
+});
```

Comment:
Should we add the flag fro preview and demo to API tests?
-> if (['preview', 'demo'].includes(config.runningEnv))

Reply:
As mentioned by Azam, this check is not needed as the API tests are available only in Preview and Demo environment.